src/GusApi/Type/GetFullReport.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace GusApi\Type;
```

Comment:
Może warto podzielić ten namespace na `Request` i `Response`?

Reply:
Te klasy zostały wygenerowane automatycznie i trzeba, poukładać jakoś sensownie i zmienić ich kod bo skrypt który je wygenerować zrobił to mało sensownie. Ze względu na brak czasu zrobiłem to bardzo pobieżnie :(